microsoft-edge/toc.yml

```
@@ -1666,6 +1666,34 @@
           href: accessibility/test.md
           displayName: 
 
+#==============================================================================
+# Dual Engine
+    - name: DualEngine
+      items:
+        - name: Introduction to Microsoft DualEngine API  # existing firstchild article
```

```suggestion
        - name: Introduction to Microsoft DualEngine API
```
comment is probably a copy/paste from another line, not relev here